requirements.txt

```diff
@@ -1,6 +1,7 @@
 django-taggit==1.2.0
 django>=2.2.11,<3.0
 django-coverage-plugin==1.8.0  # https://github.com/nedbat/django_coverage_plugin
-psycopg2==2.8.4 --no-binary psycopg2 
-Sphinx==2.4.4  
-pytest-django==3.8.0  # https://github.com/pytest-dev/pytest-django
\ No newline at end of file
+django-webpack-loader==0.7.0
+psycopg2==2.8.4 --no-binary psycopg2
```

Comment:
Can you make a separate requirements file just for testing.  Most normal users won't need to install these additions, and psycopg2 is generally better to install from the system package manger, not through pip

Reply:
Copy/pasting our slack conversation here for context:
<img width="988" alt="Screen Shot 2020-08-25 at 9 12 10 AM" src="https://user-images.githubusercontent.com/24264529/91199726-1f7c5d80-e6b3-11ea-933f-7bfcd79292c4.png">

<img width="991" alt="Screen Shot 2020-08-25 at 9 10 49 AM" src="https://user-images.githubusercontent.com/24264529/91199604-ed6afb80-e6b2-11ea-8c16-a5d3fa9c4adb.png">

Reply:
Requirements are to run tests. Tests should be runnable in scarlet without installing Scarlet Demo. This needs to be a thing

```
cd scarlet
pip install requirements ( or pipenv variant )
python manage.py test
cd docs/
make html
```

 @markmiscavage what is the advantage of using the system psycopg2 vs the pip aside from build time? Build time is certainly enough, just checking.

Reply:
Build time is the main one. The system build is also tailored to your system so the install is consistent, where as the pip build could fail because of your C compiler (or lack of one).